ports/espressif/common-hal/busio/I2C.c

```diff
@@ -73,9 +73,6 @@ void common_hal_busio_i2c_construct(busio_i2c_obj_t *self,
         mp_raise_ValueError(MP_ERROR_TEXT("All I2C peripherals are in use"));
     }
 
-    // Delete any previous driver.
-    i2c_driver_delete(self->i2c_num);
-
```

Comment:
Could you say why you removed this? Thanks.

Reply:
I was seeing extra logging that this was failing. I think init/deinit code does this correctly.